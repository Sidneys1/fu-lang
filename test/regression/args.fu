main: u8(args: str[]) = {
    return args.length;
};