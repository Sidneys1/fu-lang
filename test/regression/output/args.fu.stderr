Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "S:\fugly-compiler\fu\__main__.py", line 142, in <module>
    sys.exit(main())
             ^^^^^^
  File "S:\fugly-compiler\fu\__main__.py", line 96, in main
    return compiler_main(ns)
           ^^^^^^^^^^^^^^^^^
  File "S:\fugly-compiler\fu\compiler\main.py", line 144, in main
    binary, c_errors = collect_returning_generator(compile_binary())
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "S:\fugly-compiler\fu\compiler\util.py", line 68, in collect_returning_generator
    ret2 = list(_())
           ^^^^^^^^^
  File "S:\fugly-compiler\fu\compiler\util.py", line 66, in _
    ret = yield from generator
          ^^^^^^^^^^^^^^^^^^^^
  File "S:\fugly-compiler\fu\compiler\compile\__init__.py", line 68, in compile_binary
    main_func = compile_func(main_func_id, main)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "S:\fugly-compiler\fu\compiler\compile\__init__.py", line 228, in compile_func
    for source_loc in compile_blocks(element.initial.content, buffer):
  File "S:\fugly-compiler\fu\compiler\compile\__init__.py", line 150, in compile_blocks
    yield from compile_statement(statement, buffer)
  File "S:\fugly-compiler\fu\compiler\compile\__init__.py", line 124, in compile_statement
    return_storage = yield from compile_expression(statement.value, buffer, want=fn_ret)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "S:\fugly-compiler\fu\compiler\compile\_compile_expression.py", line 171, in compile_expression
    raise NotImplementedError()
NotImplementedError
